component/app.js

```diff
@@ -0,0 +1,40 @@
+class App {
+    constructor(gradeTable) {
+        this.gradeTable = gradeTable;
+        
+        this.handleGetGradesError = this.handleGetGradesError.bind(this);
+        this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+
+    }
+
+
+    handleGetGradesError(error) {
+        console.error(error);
+    }
+
+    handleGetGradesSuccess(grades) {
+        console.log("grades: ", grades);
+

```

Comment:
For the same of keeping your code clean, please remove any `console.log`s after you're done with them.